bigtable/client/row_reader_test.cc

```diff
@@ -797,8 +858,9 @@ TEST_F(RowReaderTest, FailedStreamRetryNewContext) {
   }
 
   parser_factory_->AddParser(std::move(parser));
+  bigtable::TableId table_id("");
```

Comment:
Why not inline the calls to `TableId`, as in:

```C++
bigtable::RowReader reader(client_, bigtable::TableId(""), 
```

Reply:
Done.